Homework_3/Homework_3.py

```diff
@@ -0,0 +1,47 @@
+# input string
+text = """homEwork:
+tHis iz your homeWork, copy these Text to variable.
+You NEED TO normalize it fROM letter CASEs point oF View.also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+it iZ misspeLLing here.fix“iZ” with correct “ is ”, but ONLY when it Iz a mistAKE.
+last iz TO calculate nuMber OF Whitespace characteRS in this Text.caREFULL, not only Spaces, but ALL whitespaces.I got 87."""
+
+normalized_text = ''
+last_words = []
+
+for x in text.lower().split('\n'):  # Convert string into lower case and split it by '\n'.
+    for y in x.split('.'):  # Split again by '.'.
+        if y.find('iz') != -1:  # If 'iz' not False.
```

Comment:
It's work good, but you can use:
`if 'iz' in y:`
It's much easier and understandable.